addons/event/event.py

```diff
@@ -303,6 +303,27 @@ def create(self, vals):
             registration.sudo().confirm_registration()
         return registration
 
+    @api.model
+    def _prepare_attendee_values(self, registration):
+        """ Method preparing the values to create new attendees based on a
+        sale order line. It takes some registration data (dict-based) that are
+        optional values coming from an external input like a web page. This method
+        is meant to be inherited in various addons that sell events.
+
+        Kwargs can be used to update the method available parameter according to
```

Comment:
This old comment can be removed I think (don't know, I think I added this one before rewriting something else).